web/src/main/java/org/akaza/openclinica/controller/openrosa/OpenRosaSubmissionController.java

```diff
@@ -150,6 +157,17 @@
         }
     }
 
+    private boolean isParticipantSubmission(HashMap<String, String> subjectContext) {
+        boolean isParticipant = true;
+        String userAccountId = subjectContext.get("userAccountID");
+        if (!StringUtils.isEmpty(userAccountId)) {
```

Comment:
Should use org.apache.commons.lang.StringUtils.isNotEmpty(str) instead.  Less dependent on Spring